website/content/docs/enterprise/fips.mdx

```diff
@@ -0,0 +1,127 @@
+---
+layout: docs
+page_title: FIPS 140-2
+description: >-
+  A version of Consul compliant with FIPS 140-2 is available to Enterprise users. Learn about where to find FIPS-compliant versions of Consul, as well as usage restrictions and technical details.
+---
+
+# FIPS 140-2
+
+<Note> This feature requires Consul Enterprise. </Note>
+
+Builds of Consul Enterprise marked with a `fips1402` feature name include built-in support for FIPS 140-2 compliance.
+
+To use this feature, you must have an [active or trial license for Consul Enterprise](/consul/docs/enterprise/license/overview). To start a trial, contact HashiCorp sales.
+
+## Using FIPS 140-2 Consul Enterprise
+
+FIPS 140-2 versions of Consul Enterprise behave like non-FIPS versions of Consul. No restrictions are placed on algorithms and it is up to the operator to ensure Consul remains in a FIPS-compliant mode of operation. Maintaining FIPS-compliant operation requires configuring some settings such as [ensuring TLS is enabled to require encrypted communication](consul/tutorials/security/tls-encryption-secure). Consul products surface some helpful warnings where settings are insecure.
```

Comment:
```suggestion
FIPS 140-2 builds of Consul Enterprise behave in the same way as non-FIPS builds. There are no restrictions on Consul algorithms and ensuring that Consul remains in a FIPS-compliant mode of operation is your responsibility. To maintain FIPS-compliant operation, you must [ensure that TLS is enabled](consul/tutorials/security/tls-encryption-secure) so that communication is encrypted. Consul products surface some helpful warnings where settings are insecure.
```